benchmarks/decoders/benchmark_decoders_library.py

```diff
@@ -239,6 +261,12 @@ def __init__(self, num_ffmpeg_threads=None, device="cpu"):
         )
         self._device = device
 
+        import torchvision  # noqa: F401
+        from torchvision.transforms import v2 as transforms_v2
+
+        self.torchvision = torchvision
```

Comment:
I don't think we actually use `self.torchvision`? We should be able to remove it.